packages/ecr-credential-provider/ecr-credential-provider.spec

```diff
@@ -0,0 +1,48 @@
+%global goproject github.com/kubernetes
+%global gorepo cloud-provider-aws
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.25.1
+%global rpmver %{gover}
+
+%global _dwz_low_mem_die_limit 0
+
+%global gitrev 704b05de2c8633e4acaae62bd81c5575e1e5c1d6
+%global shortrev %(c=%{gitrev}; echo ${c:0:7})
+
+Name: %{_cross_os}aws-ecr-credential-provider
```

Comment:
Can we name this RPM `ecr-credential-provider` to match the binary name and the spec file?

Reply:
Done!